### PR TITLE
Trigger

### PR DESCRIPTION
**Pull request by evancohen**
@ashishsc 

Adding sonus hotword trigger and documentation that I wrote on the plane :)
Wasn't sure if this was really the right approach for the trigger, let me know what you think.

### CODE REVIEW

File: docs/API.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+`sensitivity` - (default `'0.5'`) If you are getting a lot of false positives or are having trouble detecting your hotword adjusting this value shoud help
+
+**Example:** (to be passed into the sonus constructor)  
+``` javascript
+const hotwords = [
+    {file: '/mymodel.pmdl', hotword: 'sonus'},
+    {file: 'snowboy.umdl', hotword: 'snowboy'}]
+```
+
+### Lenguages
```
Comment (jhoelzl) [inline docs/API.md]: Languages

File: index.js [hunk 2]
```diff
@@ -97,6 +101,16 @@ Sonus.init = (options, recognizer) => {
       }
     }
   })
+
+  sonus.trigger = (index, hotword) => {
+    if(sonus.started){
+      sonus.emit('hotword', index || "0", hotword || "triggered")
```
Comment (ashishsc) [inline index.js]: Did you actually want string 0?

Comment (evancohen) [inline index.js]: Ping!

Comment (ashishsc) [inline index.js]: You used a string

Comment (evancohen) [inline index.js]: Yeah - it's actually supposed to be a string... My proposal was wrong.

File: docs/API.md [hunk 3]
```diff
@@ -80,21 +80,31 @@ Sonus.stop(sonus)
 Note that after recognition is stopped it can not be started again without creating an enterly new sonus instance.
 
 ### Trigger keyword/hotword manually
-You can manuall trigger a hotword by passing your initialized sonus object into `Sonus.trigger`
-This will throw a `NOT_STARTED` exception if you have not started sonus when this is called. 
+You can manuall trigger a hotword by passing your initialized sonus object and an index into `Sonus.trigger`
```
Comment (ashishsc) [inline docs/API.md]: typo

File: index.js [hunk 4]
```diff
@@ -104,8 +105,14 @@ Sonus.init = (options, recognizer) => {
...
   sonus.trigger = (index, hotword) => {
     if(sonus.started){
-      sonus.emit('hotword', index || "0", hotword || "triggered")
-      CloudSpeechRecognizer.startStreaming(opts, sonus.mic, csr)
+      try{
+        let triggerHotword = (index == 0)? hotword : models.lookup(index)
+        sonus.emit('hotword', index, triggerHotword)
+        CloudSpeechRecognizer.startStreaming(opts, sonus.mic, csr)
+      } catch (e) {
+        console.log(e)
```
Comment (ashishsc) [inline index.js]: console.error

**Pull request merged**